pkg/fleet/bootstraper/bootstraper.go

```diff
@@ -152,5 +153,18 @@ func Bootstrap(ctx context.Context, opts ...Option) error {
 	if err != nil {
 		return fmt.Errorf("failed to write hash file: %w", err)
 	}
+
+	// 6. Get the list of default packages to install and install them.
+	defaultPackages, err := cmd.DefaultPackages(ctx)
```

Comment:
I think it's safer to do 6 as a single command in the installer. Bootstrap and Installer version can differ, and make later changes complicated.

Example here the packages can have 1 version {packageName:version}, but it's possible that we'll want to install multiple tracers of a same package in a near future.

Maybe a command InstallDefaults on the installer that self re-execute DefaultPackages + per package install?

Reply:
I'll move it. I reached the current solution because I don't want to introduce two mode of install. All installation should happen through a `datadog-installer install`. On paper It's also fine to do bootstraper => `datadog-installer install-default-packages` => N time `datadog-installer <package>`.

Reply:
I ended up trying a simpler approach:
- DefaultPackage resolves the URLs directly
- The bootstraper simply iterates over all the URLs